cli/operator/node.go

```diff
@@ -166,7 +163,8 @@ var StartNodeCmd = &cobra.Command{
 		cfg.P2pNetworkConfig.Permissioned = permissioned
 		cfg.P2pNetworkConfig.WhitelistedOperatorKeys = append(cfg.P2pNetworkConfig.WhitelistedOperatorKeys, networkConfig.WhitelistedOperatorKeys...)
 		cfg.P2pNetworkConfig.NodeStorage = nodeStorage
-		cfg.P2pNetworkConfig.OperatorID = format.OperatorID(operatorData.PublicKey)
+		cfg.P2pNetworkConfig.OperatorPublicKey = format.OperatorID(operatorData.PublicKey)
```

Comment:
@nkryuchkov why we change the name if its OperatorID?

Reply:
@lior-blox it's not numeric operator ID, which I also need to have in P2pNetworkConfig. But since it's hash of public key, I'll rename it to `OperatorPublicKeyHash` or `OperatorIdentity`

Reply:
fixed